lib/webhooks/process.ts

```diff
@@ -68,7 +69,13 @@ export function process(
           log.debug('Found HTTP handler, triggering it', loggingContext);
 
           try {
-            await handler.callback(graphqlTopic, domain, rawBody, webhookId);
+            await handler.callback(
```

Comment:
The number of arguments here is getting a little bit out of hand, it'd be nice to make this an object in a future breaking change.

Reply:
I was thinking the same thing while making the change!